spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,42 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def create_control_layout(controls, main_widget):
+    """
+    Returns a layout for a set of controls above a main widget. This is a
+    standard layout for many panes and some other widgets.
+
+    The use-case is currently limited to a single line of controls.
+
+    controls: a list of QWidgets to be placed in a HBoxLayout at the top.
+        You may use 'stretch' to add a stretch to the layout.
+        If a more complex control of the layout is needed, you can pass a
+        ready made layout instead.
+    main_widget: the main widget. Can be None, if you want to add this
+        manually later on.
+    """
+    layout = QVBoxLayout()
+    layout.setContentsMargins(0, 0, 0, 0)
+    layout.setSpacing(2)

```

Comment:
Yes it should be OS centric, and you could use http://doc.qt.io/qt-5/qstyle.html#PixelMetric-enum